.gitignore

```diff
@@ -57,3 +57,9 @@ cmake-build-debug/
 cmake-build-release/
 
 test/models/
+
+# Emacs intermediate files
+*~
+
+# CMake building directory
+build
```

Comment:
# CMake building directory
build
cmake-build-debug/
cmake-build-release/

Reply:
The other two directories are the default construction directories for clion.

Reply:
The other two directories are the default build directories for clion

Reply:
Done